File: Iniciante/URI 1930.go
package main
 
import (
    "fmt"
)
 
func main() {
    var T1, T2, T3, T4, totalTomadas int

    fmt.Scanf("%d", &T1)
    fmt.Scanf("%d", &T2)
    fmt.Scanf("%d", &T3)
    fmt.Scanf("%d", &T4)
	
    totalTomadas = (T1 + T2 + T3 + T4) - 3
	
    fmt.Println(totalTomadas)
}

File: Iniciante/URI 1930.ml
let t1, t2, t3, t4 = Scanf.scanf "%i %i %i %i" (fun a b c d -> a, b, c, d);;
let totalTomadas = (t1 + t2 + t3 + t4) - 3;;
Printf.printf "%d\n" (totalTomadas);